pkg/cvo/metrics.go

```diff
@@ -164,6 +186,20 @@ func RunMetrics(runContext context.Context, shutdownContext context.Context, lis
 					klog.Errorf("Collected metrics %s goroutine: %v", result.name, result.error)
 					loopError = result.error
 				}
+			case event := <-watcher.Events:
+				klog.V(2).Infof("!!!! Event from filewatcher on file: %v, event: %v", event.Name, event.Op)
+
+				if event.Op == fsnotify.Chmod || event.Op == fsnotify.Remove {
+					continue
+				}
+
+				if certsChanged(origCertChecksum, origKeyChecksum, certFile, keyFile) {
+					// Update file checksums with latest files.
+					origCertChecksum = checksumFile(certFile)
+					origKeyChecksum = checksumFile(keyFile)
```

Comment:
we're about to fall through into the shutdown loop.  Do we need to update these checksum variables?

Reply:
Probably so.. Still in work...

Reply:
makes sense to me to do this, now that you're just restarting the metrics server, and not restarting the whole container.